config/locales/views/en.yml

```diff
@@ -375,6 +375,10 @@ en:
           disable_anon: Sorry, this work doesn't allow non-Archive users to comment.
           hidden: Sorry, you can't add or edit comments on a hidden work.
           unrevealed: Sorry, you can't add or edit comments on an unrevealed work.
+  downloads:
+    download_afterword:
+      please_comment_html: Please %{work_comment_link} to let the creator know if you enjoyed their work!
+      work_comment: drop by the archive and comment
```

Comment:
(not AD&T) Since we're editing it, I think it would be good to capitalise the "Archive" as a proper noun.

Reply:
Good point, the newest texts indeed capitalize it. If AD&T says that it's wrong it can be changed back.